metadata-v2.graphqls

```diff
@@ -22,27 +22,31 @@
 
 # Service is a logic concept, representing a collection of runnable context.
 type Service {
+    # Service ID = BASE64(name) + '.' + Layer ID
```

Comment:
Now this is totally incompatible right?

Reply:
For existing data, yes, it isn't. Because one way or another, we need layer definition for service to visualize in the booster UI.